.github/workflows/buildAndTest.yml

```diff
@@ -38,8 +39,16 @@ jobs:
           # Exclude macos-arm64 and llvm out-of-tree altogether
           - os-arch: macos-arm64
             llvm-build: out-of-tree
+          - os-arch: macos-arm64
+            torch-version: stable
           - os-arch: windows-x86_64
             llvm-build: out-of-tree
+          - os-arch: windows-x86_64
+            torch-version: stable
+          - os-arch: ubuntu-x86_64
+            llvm-build: out-of-tree
+          - os-arch: ubuntu-x86_64
+            torch-version: nightly
```

Comment:
Is this intended ? We will only build on stable if we exclude nightly right ?

Reply:
By default, every combination of parameters is activated. We have to actively disable some of them. Here I removed some of the Linux runs to save build time. I will reactivate them to avoid confusion.